spec/granite/querying/query_builder_spec.cr

```diff
@@ -3,6 +3,22 @@ require "../../spec_helper"
 describe Granite::Query::BuilderMethods do
   describe "#where" do
     describe "with array argument" do
+      it "errors queries ids fields" do
+        review1 = Review.create(name: "one")
+        review2 = Review.create(name: "two")
+
+        found = Review.where(id: [review1.id, review2.id]).select
+        found[0].id.should eq review2.id
+        found[1].id.should eq review1.id
+      end
+
+      it "correctly queries with ids fields which doest exists" do
+        Review.create(name: "one")
+        Review.create(name: "two")
+
+        found = Review.where(id: [1001, 1002]).select
+        found.size.should eq 0
+      end
       it "correctly queries string fields" do
```

Comment:
Add a test for `and` and `or` operators.

Reply:
done